Unify expectations writing #1778 (#1833)
act-rules/act-rules.github.io: _rules/audio-as-media-alternative-afb423.md

## Expectation 2

> Each target element is labeled as a video alternative for text on the page by a [visible][] identifier that is [included in the accessibility tree][].

Reviewer: I'm not a huge fan of breaking the conjuction, it makes it a bit less clear that both conditions are needed.
Maybe:
```suggestion
Each target element is labeled as a video alternative for text on the page by an identifier that is [visible][] and [included in the accessibility tree][].
```
(on all 3 rules)

> Each target element is labeled as a video alternative for text on the page by an identifier that is [visible][] and [included in the accessibility tree][].

Reviewer: ```suggestion
Each target element is labeled as an audio alternative for text on the page by an identifier that is [visible][] and [included in the accessibility tree][].
```
That one rule is about audio.

> **Note:** The term label does not refer to the `label` element.

Reviewer: ```suggestion
```
No more label in the expectation 🎉
